ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -219,6 +219,37 @@ public void setDownstreamHttpPath(final String path) {
       defaultValue = "localhost,127.0.0.1")
   private final AllowListHostsProperty metricsHostAllowList = new AllowListHostsProperty();
 
+  @SuppressWarnings("FieldMayBeFinal") // Because PicoCLI requires Strings to not be final.
+  @Option(
+      names = {"--http-proxy-host"},
+      description = "Hostname for proxy connect, no proxy if null (default: null)",
```

Comment:
Using ${DEFAULT-VALUE} create conflicts during tests at this line :
`assertThat(commandOutput.toString()).containsOnlyOnce(defaultUsageText);`
Because defaultUsageText is populated with CmdlineHelpers value.
For now, I think it is more simple to not use it, being consistent with downstream-http-request-timeout